readme.txt

```diff
@@ -6,6 +6,7 @@
 
 * Tweak - Extended support for namespaced classes in the Autoloader.
 * Tweak - Normalize namespaced prefixes with trailing backslash when registering them in the Autoloader. [VE-14]
+* Tweak - Fix where a text settings field would show up empty if the saved value is '0' [BTRIA-338]
```

Comment:
```suggestion
* Tweak - Allow text fields to display strings that evaluate to empty, such as a zero string (`'0'`). Useful for a setting asking what the display value of a free ticket should be. [BTRIA-338]
```

This will need to make it out of triage before allowing the ticket number in the changelog.